app/assets/stylesheets/views/_covid.scss

```diff
@@ -123,6 +123,9 @@
   @include govuk-responsive-margin(9, "bottom");
   background-color: $covid-grey;
   padding: govuk-spacing(6) govuk-spacing(5);
+  border-left: 10px solid;
+  border-width: 0 0 0 10px;
```

Comment:
Toggling this line in the browser inspector doesn't seem to change anything - is this for older browsers?

Reply:
your right - sorry I forgot to remove, the `border-left` has overridden it